modules/storages/app/services/storages/storages/set_attributes_service.rb

```diff
@@ -39,14 +39,24 @@ def set_default_attributes(_params)
     private
 
     def set_attributes(params)
-      super(params)
+      super(replace_empty_host_with_nil(params))
```

Comment:
Why do we have to replace it? And do we really have to clone? Can we modify params, by removing the key, for instance?

Reply:
Same answer as above ;)